### PR TITLE
fix: typo in Target the Parent of an Element Using jQuery module

### LINKED ISSUE #50127: Typo in Target the Parent of an Element Using jQuery module
### Describe the Issue

In [Target the Parent of an Element Using jQuery](https://www.freecodecamp.org/learn/front-end-development-libraries/jquery/target-the-parent-of-an-element-using-jquery) module, currently, the second paragraph happens to be as follows:

For example, your `jQuery Playground` `h3` element has the parent element of `<div class="container-fluid">`, which itself has the parent `body`.

For the sake of clarity, I think the instruction should be as follows:

For example, the `h3` element in your `jQuery Playground` has the parent element of `<div class="container-fluid">`, which itself has the parent element of `body`.

### Affected Page

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/03-front-end-development-libraries/jquery/target-the-parent-of-an-element-using-jquery.md

### Expected behavior

The second paragraph in the [Target the Parent of an Element Using jQuery](https://www.freecodecamp.org/learn/front-end-development-libraries/jquery/target-the-parent-of-an-element-using-jquery) module, should be as follows:

For example, the `h3` element in your `jQuery Playground` has the parent element of `<div class="container-fluid">`, which itself has the parent element of `body`.

### Screenshots

image

### System

- Device: Laptop
- OS: macOS 13.0.1
- Browser: Google Chrome
- Version: 107.0.5304.110

### PR DESCRIPTION
**Pull request by anmxls**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #50127

### CODE REVIEW

**Review by Sembauke — APPROVED** [review]

**Comment by lasjorg** [comment]
Does it need the first two sentences?

I feel like the first is irrelevant and the second is implied by what the challenge is teaching.

File: curriculum/challenges/english/03-front-end-development-libraries/jquery/target-the-parent-of-an-element-using-jquery.md
Comment (lasjorg) [reply curriculum/challenges/english/03-front-end-development-libraries/jquery/target-the-parent-of-an-element-using-jquery.md:13]: I think we mainly use the element name version without the `<>` in the curriculum.

**Comment by lasjorg** [comment]
Personally, I don't see the two sentences providing much, if any, context to the challenge. The first seems superfluous and the next gives an example before we again give an example.

Anyway, I'm fine with keeping them. I will approve the PR but not merge it, so we still have the option to remove them if someone else thinks they should be removed, or just do the merge.

**Review by lasjorg — APPROVED** [review]
LGTM if no one else wants that we remove the first two sentences.

File: curriculum/challenges/english/03-front-end-development-libraries/jquery/target-the-parent-of-an-element-using-jquery.md
Comment (lasjorg) [reply curriculum/challenges/english/03-front-end-development-libraries/jquery/target-the-parent-of-an-element-using-jquery.md:11]: I agree, we should remove the backticks (if we keep the sentence) and I don't think the words really need any emphasis.

**Pull request merged**
